tests/test_plots.py

```diff
@@ -246,3 +272,52 @@ def test_plot_ridge_aes_labels_shading(self, backend, datatree_4d, pseudo_dim):
         if pseudo_dim != "__variable__":
             assert all(0 in child["alpha"] for child in pc.aes.children.values())
             assert any(pseudo_dim in child["shade"].dims for child in pc.viz.children.values())
+
+    @pytest.mark.parametrize("group", ("prior", "posterior"))
+    @pytest.mark.parametrize("kind", ("kde", "cumulative"))
+    def test_plot_ppc(self, datatree, kind, group, backend):
+        pc = plot_ppc(datatree, kind=kind, group=group, backend=backend)
+        assert "chart" in pc.viz.data_vars
+        assert "obs" in pc.viz
+        assert "ppc_dim" in pc.viz["obs"].dims
+        if kind == "kde":
+            assert "kde" in pc.viz["obs"]
+        elif kind == "cumulative":
+            assert "ecdf" in pc.viz["obs"]
+        assert "overlay" in pc.aes["obs"].data_vars
+
+    @pytest.mark.parametrize("group", ("prior", "posterior"))
+    @pytest.mark.parametrize("kind", ("kde", "cumulative"))
+    def test_plot_ppc_sample(self, datatree_sample, kind, group, backend):
+        pc = plot_ppc(
+            datatree_sample, kind=kind, group=group, sample_dims="sample", backend=backend
+        )
+        assert "chart" in pc.viz.data_vars
+        assert "obs" in pc.viz
+        assert "ppc_dim" in pc.viz["obs"].dims
+        if kind == "kde":
+            assert "kde" in pc.viz["obs"]
+        elif kind == "cumulative":
+            assert "ecdf" in pc.viz["obs"]
+        assert "overlay" in pc.aes["obs"].data_vars
+
+    @pytest.mark.parametrize("group", ("prior", "posterior"))
+    @pytest.mark.parametrize("kind", ("kde", "cumulative"))
+    @pytest.mark.parametrize("facet_dims", (["group"], ["hierarchy"], None))
+    def test_plot_ppc_4d(self, datatree_4d, facet_dims, kind, group, backend):
```

Comment:
there should be assertions to check `facet_dims` is working as expected. For example, making sure that dimension is present in `pc.viz["obs"]["plot"]`